inference-engine/tests/ie_test_utils/functional_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -187,6 +187,38 @@ void compare(
 #undef CASE0
 #undef CASE
 }
+
+void set_dimension_intervals(std::vector<ov::test::InputShape>& inputShapes) {
+    for (auto& input_shape : inputShapes) {
+        const auto model_dynamic_shape = input_shape.first;
+        if (!model_dynamic_shape.is_dynamic()) {
+            throw ov::Exception("input shape is not dynamic");
```

Comment:
It would be better to add more context to the error message:
"unexpected static shape in set_dimension_intervals" or something like that

Reply:
will do, thanks